### PR TITLE
Mimic intellij github copilot plugin supported short cuts

### PR DESCRIPTION
**Pull request by tninja**
Also added a [transient](https://github.com/magit/transient) based menu. (Initially by ancilla-generate)

Screenshot 2024-09-06 at 5 52 04 PM

also remove a dup entry

@shouya PTAL Thanks

### CODE REVIEW

**Comment by tninja** [comment]
Current intellij github copilot plugin menu looks like

Screenshot 2024-09-06 at 5 55 58 PM

**Comment by shouya** [comment]
Thanks, that looks quite useful! Can you update README about this feature as well?

File: ancilla.el [hunk 1]
```diff
@@ -111,9 +112,61 @@ call `ancilla-rewrite' otherwise."
   (interactive)
   (ancilla--call-adaptor-with-instruction 'rewrite))
 
-;; ------------ PRIVATE FUNCTIONS --------------
+;; shortcuts following intellij copilot plugin
+
+(defun ancilla-simplify-this ()
+  "Simplify the current selection."
+  (interactive)
+  (ancilla--process-rewrite "Simplify this"))
```
Comment (shouya) [inline ancilla.el:120]: how about let the user to customize the prompts by prefixing `C-u`?

Comment (tninja) [reply ancilla.el:120]: yes. i wish to have a better prompts than this.. for more user specific simplify operation, i think ancilla-rewrite is a better option.

Comment (shouya) [reply ancilla.el:120]: Make sense. Let's keep i this way then. Ultimately I think it may be a good idea to let the user to customize the shortcuts.

**Review by shouya — APPROVED** [review]
Great job!

**Pull request merged**